scan-manager/docs/setup.md

```diff
@@ -26,9 +26,9 @@
 - Default `config.yaml` is provided inside `setup_artifacts/deployment` directory for initial setup
 - [Add](#adding-templates) corresponding templates for the given model types and base-image
 - Make sure to update the `license.txt` file for h2o-mojo in the `setup_artifacts/files` directory (when working with h2o-mojo models)
-- Run the `sh upload_artifact.sh <AWS_BUCKET>` script from the `setup_artifacts` directory   
-  to make the updated artifacts available for Scan Manager to get started
-- `AWS_BUCKET` is the same bucket configured when installing Scan Manager
+- Run the `sh upload_artifact.sh <END_POINT> <ACCESS_KEY> <SECRET_KEY> <BUCKET_NAME>` script from the `setup_artifacts` directory   
+  to make the updated artifacts available for Scan Manager to get started[Minio Client should be installed before running sh upload_artifact.sh]
```

Comment:
One thing I've noticed is with `sh <script>`, it doesn't use `bash` features and results in errors. I hope that's not the case with this script. Is there a reason we're using `sh` instead of `bash`?
Probably a question for @akumar-cs

Reply:
not really, i hadn't thought about it that much tbh.

Reply:
Please change to bash not sh

Reply:
changed to bash